op-challenger/game/fault/preimages/direct.go

```diff
@@ -38,23 +38,15 @@ func (d *DirectPreimageUploader) UploadPreimage(ctx context.Context, claimIdx ui
 	if err != nil {
 		return fmt.Errorf("failed to create pre-image oracle tx: %w", err)
 	}
-	if err := d.sendTxAndWait(ctx, candidate); err != nil {
+	if err := d.sendTxAndWait(candidate); err != nil {
 		return fmt.Errorf("failed to populate pre-image oracle: %w", err)
 	}
 	return nil
 }
 
 // sendTxAndWait sends a transaction through the [txmgr] and waits for a receipt.
 // This sets the tx GasLimit to 0, performing gas estimation online through the [txmgr].
-func (d *DirectPreimageUploader) sendTxAndWait(ctx context.Context, candidate txmgr.TxCandidate) error {
-	receipt, err := d.txMgr.Send(ctx, candidate)
-	if err != nil {
-		return err
-	}
-	if receipt.Status == ethtypes.ReceiptStatusFailed {
-		d.log.Error("DirectPreimageUploader tx successfully published but reverted", "tx_hash", receipt.TxHash)
-	} else {
-		d.log.Debug("DirectPreimageUploader tx successfully published", "tx_hash", receipt.TxHash)
-	}
-	return nil
+func (d *DirectPreimageUploader) sendTxAndWait(candidate txmgr.TxCandidate) error {
+	_, err := d.txSender.SendAndWait(candidate)
```

Comment:
Should we log the tx receipt here?

Reply:
The txSender logs when the tx is successful in the same way the old code did. That's actually the majority of the code we kept duplicating.